includes/class-wc-cache-helper.php

```diff
@@ -107,7 +107,7 @@ public static function get_cache_prefix( $group ) {
 	 * @param string $group Group of cache to clear.
 	 */
 	public static function incr_cache_prefix( $group ) {
-		wp_cache_incr( 'wc_' . $group . '_cache_prefix', 1, $group );
+		wp_cache_set( 'wc_' . $group . '_cache_prefix', microtime(), $group );
```

Comment:
Now that we are not incrementing the cache prefix anymore, I would suggest updating the method docblock to reflect this change.

The method name also indicates that it increments the cache prefix, but changing it is a bit more involved since this is a public method. I see two possible solutions in this case. We simply add a note to the docblock explaining why the method name is incorrect or we deprecate and change it to call a new method. Something like:

```
public static function incr_cache_prefix( $group ) {
	wc_deprecated_function( 'WC_Cache_Helper::incr_cache_prefix', '3.9.0', 'WC_Cache_Helper::create_new_cache_prefix' );
	self::create_new_cache_prefix( $group );
}

public static function create_new_cache_prefix( $group ) {
	wp_cache_set( 'wc_' . $group . '_cache_prefix', microtime(), $group );
}
```

What do you think?

Reply:
I like the deprecation approach seeing as there is no incrementing happening anymore.

Reply:
I like the suggestion. I changed the new function's name to hopefully better reflect its conceptual role instead of the actual implementation.